source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,39 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-06-30
+
+### Highlights
+
+### Unlimited E10/20 trials
+ - Users are now prohibited from starting unlimited E10/20 trials.
+
+### Emoji Enhancements with Skin Tone Selection
+ - Added support for emoji standard v13.0. Users will see Reactions and Emoji rendered based on their operating system or device, and have the ability to choose various skin tones using the Mattermost emoji picker.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added English-Australian English language variation.
+ - Added the ability to upload ``.jpeg`` files on Linux.
```

Comment:
question: are both .jpeg and .jpg files supported? If so, consider adding this level of detail to the changelog

Reply:
.jpeg files were already supported, I added a note.